foxpuppet/browser/window.py

```diff
@@ -2,43 +2,76 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from foxpuppet.windows import Windows
-
 from selenium.webdriver.common.by import By
 
+from foxpuppet.windows import BaseWindow
+
 
-class BrowserWindow(object):
+class BrowserWindow(BaseWindow):
     _file_menu_button_locator = (By.ID, 'file-menu')
     _file_menu_private_window_locator = (By.ID, 'menu_newPrivateWindow')
     _file_menu_new_window_button_locator = (By.ID, 'menu_newNavigator')
     _nav_bar_locator = (By.ID, 'nav-bar')
     _tab_browser_locator = (By.ID, 'tabbrowser-tabs')
 
     def __init__(self, selenium, *args, **kwargs):
+        super(BrowserWindow, self).__init__(selenium)
         self.selenium = selenium
```

Comment:
There is no need that each and every subclass has to set the selenium property. It should be done by the base window class only.